elasticjob-lite/elasticjob-lite-core/src/test/java/org/apache/shardingsphere/elasticjob/lite/internal/config/RescheduleListenerManagerTest.java

```diff
@@ -63,19 +63,19 @@ public void assertStart() {
     @Test
     public void assertCronSettingChangedJobListenerWhenIsNotCronPath() {
         rescheduleListenerManager.new CronSettingAndJobEventChangedJobListener().dataChanged("/test_job/config/other", Type.NODE_CREATED, LiteYamlConstants.getJobYaml());
-        verify(jobScheduleController, times(0)).rescheduleJob(ArgumentMatchers.any());
+        verify(jobScheduleController, times(0)).rescheduleJob(ArgumentMatchers.any(), ArgumentMatchers.any());
```

Comment:
Use static import for `ArgumentMatchers`.

Reply:
ok